src/libs/actions/IOU.js

```diff
@@ -154,6 +154,7 @@ function buildOnyxDataForMoneyRequest(
             value: {
                 ...chatReport,
                 lastReadTime: DateUtils.getDBTime(),
+                lastMessageTranslationKey: '',
```

Comment:
Why is this needed in this PR?

Reply:
I added it as mentioned [here](https://github.com/Expensify/App/pull/28244#issuecomment-1738672824)

Reply:
Oh, sorry. That's that key.